docs/api-table.js

```diff
@@ -11,24 +11,24 @@ export const react = {
   Button: {
     required: [],
     props: [
-      ['label', 'string', '', 'Interchangeable with the default slot for labelling'],
-      ['href', 'string', '', 'When set, an anchor tag will be used instead of a button'],
-      ['type', 'string', 'button', "Controls the button's type, unused when `href` is present"],
-    ],
-    slots: [
-      ['default', 'can be used instead of the label prop for button content']
+      ['className', 'string', '', 'Additional classes to include'],
+      ['onClick', '(e: MouseEvent<HTMLButtonElement, MouseEvent>) => void', '', 'Action to be called when the component is clicked'],
+      ['style', 'CSSProperties', '', 'CSS styles to inline on the component'],
+      ['type', '|"button"|"submit"|"reset"', 'button', "Button type, only applied when href is not set."],
```

Comment:
I think we have one too many `|`, remove the first one before "button"?

<img width="360" alt="Screenshot 2023-04-06 at 17 56 09" src="https://user-images.githubusercontent.com/6739459/230433714-309571ca-42d3-43bd-b32b-ee81717e340f.png">